### PR TITLE
Remove .! special case and just consider the general case .<cmd>

### PR DESCRIPTION
**Pull request by ret2libc**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Make rz_core_cmd_str and rz_core_cmd_raw set core->is_pipe = true to
ensure that even `!` commands are piped to RzCons, so that we can get
the result later.

**Test plan**

Check build-static tests

**Closing issues**

...

### CODE REVIEW

File: librz/core/cmd.c [hunk 1]
```diff
@@ -6442,10 +6442,14 @@ static ut8 *core_cmd_raw(RzCore *core, const char *cmd, int *length) {
...
 	rz_cons_push();
+	bool is_pipe = core->is_pipe;
+	core->is_pipe = true;
 	if (rz_core_cmd(core, cmd, 0) == -1) {
+		core->is_pipe = is_pipe;
 		rz_cons_pop();
 		return NULL;
 	}
+	core->is_pipe = is_pipe;
 	rz_cons_filter();
```
Comment (wargio) [inline librz/core/cmd.c:6453]: nice catch

**Review by wargio — APPROVED** [review]

**Comment by ret2libc** [comment]
Seems the failing test is fixed on alpine. There are anyway many others...

**Comment by XVilka** [comment]
I wonder where from there is a 5 minutes difference on decoding the time:
```
[XX] db/cmd/cmd_pt ptd print ms-dos time
RZ_NOPLUGINS=1 rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc ' b 0x100 ; ptd' bins/elf/ls
-- stdout
--- expected
+++ actual
@@ -1,64 +1,64 @@
-Fri Jan 30 02:03:38 2105
-Sat Oct 10 05:41:34 2048
-Thu Oct  8 11:54:34 2048
-Sat Apr  3 09:07:04 2094
-Tue Feb 20 10:07:32 2018
-Thu Mar  6 00:44:26 1986
-Sat Oct  8 00:00:02 2050
-Tue Aug 12 05:56:26 1980
-Mon Dec 13 09:00:00 2010
+Fri Jan 30 02:08:38 2105
+Sat Oct 10 05:47:34 2048
+Thu Oct  8 11:58:34 2048
+Sat Apr  3 09:13:04 2094
+Tue Feb 20 10:09:32 2018
+Thu Mar  6 00:48:26 1986
```
cc @officialcjunior

**Review by XVilka — APPROVED** [review]

**Comment by wargio** [comment]
> I wonder where from there is a 5 minutes difference on decoding the time:
> 
> ```
> [XX] db/cmd/cmd_pt ptd print ms-dos time
> RZ_NOPLUGINS=1 rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc ' b 0x100 ; ptd' bins/elf/ls
> -- stdout
> --- expected
> +++ actual
> @@ -1,64 +1,64 @@
> -Fri Jan 30 02:03:38 2105
> -Sat Oct 10 05:41:34 2048
> -Thu Oct  8 11:54:34 2048
> -Sat Apr  3 09:07:04 2094
> -Tue Feb 20 10:07:32 2018
> -Thu Mar  6 00:44:26 1986
> -Sat Oct  8 00:00:02 2050
> -Tue Aug 12 05:56:26 1980
> -Mon Dec 13 09:00:00 2010
> +Fri Jan 30 02:08:38 2105
> +Sat Oct 10 05:47:34 2048
> +Thu Oct  8 11:58:34 2048
> +Sat Apr  3 09:13:04 2094
> +Tue Feb 20 10:09:32 2018
> +Thu Mar  6 00:48:26 1986
> ```

if this comes from Alpine, i think we need to set the alpine image to UTC.

**Pull request merged**